modules/accounts/contacts.nix

```diff
@@ -0,0 +1,70 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.accounts.contact;
+
+  contactOpts = { name, config, ... }: {
+    options = {
+      name = mkOption {
+        type = types.str;
+        readOnly = true;
+        description = ''
+          Unique identifier of the contact. This is set to the
+          attribute name of the contact configuration.
+        '';
+      };
+
+      path = mkOption {
+        type = types.str;
+        default = "${cfg.basePath}/${name}";
+        description = "The path of the storage.";
+      };
+    };
+
+    config = mkMerge [
+      {
+        name = name;
+        khal.type = mkOptionDefault "birthdays";
```

Comment:
Could be moved to the `khal-accounts.nix` file?

Reply:
The problem here is that `type` logically shouldn’t even be an option for `contact` accounts because `"birthdays"` is the only valid value. To be honest, I don’t know what to do about this; splitting `khal-accounts` in two for calendars and contacts seems cumbersome..